nncf/quantization/algorithms/post_training/backend.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2023 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC
+from abc import abstractmethod
+from pathlib import Path
+from typing import Any, Dict, List
+
+from nncf.common.graph.graph import NNCFNode
+from nncf.common.graph.transformations.commands import Command
+from nncf.quantization.algorithms.post_training.algorithm import TModel
+
+
+class PostTrainingBackend(ABC):
+    @property
+    @abstractmethod
+    def if_node_metatype(self):
```

Comment:
Maybe it's better to use `metatypes` and a list of them. But, it's minor.

Reply:
Done